manifests/a/AnyDeskSoftwareGmbH/AnyDesk/7.0.0/AnyDeskSoftwareGmbH.AnyDesk.installer.yaml

```diff
@@ -0,0 +1,39 @@
+# Created with YamlCreate.ps1 v2.0.0 $debug=AUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: AnyDeskSoftwareGmbH.AnyDesk
+PackageVersion: 7.0.0
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- Architecture: x86
+  InstallerType: msi
+  InstallerUrl: https://download.anydesk.com/AnyDesk.msi
+  InstallerSha256: 1082577C084849C0E284F80E75A444696E80F4EBB95226DA4AE5CC78837CD6C7
+  ProductCode: '{62853EBF-E9DD-4AA5-B20A-5A6C3DD74FF3}'
+  UpgradeBehavior: uninstallPrevious
+  AppsAndFeaturesEntries:
+  - DisplayName: AnyDesk MSI
+  - DisplayVersion: 7.0.0
+- Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://download.anydesk.com/AnyDesk.exe
+  InstallerSha256: C77496E2DBD3EEB440FF001FFC535A5AC9982DCADAB0FA1CBBC1EC0A9C9B7DC9
+  InstallerSwitches:
+    Silent: --silent
+    SilentWithProgress: --silent
+    Upgrade: --remove-first
+    Custom: --install "C:\Program Files (x86)\AnyDesk" --update-auto --create-desktop-icon --create-shortcuts
+  UpgradeBehavior: install
+  ElevationRequirement: elevationRequired
+  AppsAndFeaturesEntries:
+  - DisplayName: AnyDesk
+  - DisplayVersion: ad 7.0.0
```

Comment:
The client doesn't know how to match with this data yet. It can validate that it looks correct but doesn't use it for matching, which probably means that the non-MSI variant will not upgrade with this manifest.

I'll still approve if you think that's okay.

Reply:
Well thats bad news, do you know of a ETA? the 1.1 and 1.2 preview client seems to be rolling out so hopefully its not 2022<

https://github.com/microsoft/winget-pkgs/pull/35146 also got approved because i played with the ElevationRequirement variable, but it dosn't do anything on the 1.2 preview client yet..

Since you already approved the Deletion PR we have to do something about this PR tho, could probably use it for testing

Reply:
That was an oops on my part, I'm going to approve this.

@denelon is there a ETA on the implementation of https://github.com/microsoft/winget-cli/issues/1073 yet?

Reply:
Also, where did you get the 1.2 preview client? Microsoft Store?

Reply:
It's available as an update in the Microsoft Store if you're in the Dev Channel.

Reply:
It DOES say client version 1.1 tho, so its probably a different «issue» then simply supporting the variables…

And yeah kinda silly that 1.2 is delivered via MS Store with no GitHub release

Reply:
AnyDesk MSI
```
PS C:\Users\Esco> winget list anydesk
Name    Id                          Version Source
---------------------------------------------------
AnyDesk AnyDeskSoftwareGmbH.AnyDesk 7.0.0   winget
```

AnyDesk EXE
```
PS C:\Users\Esco> winget list anydesk
Name    Id                          Version  Available Source
-------------------------------------------------------------
AnyDesk AnyDeskSoftwareGmbH.AnyDesk ad 7.0.0 7.0.0     winget
PS C:\Users\Esco> winget upgrade
Name             Id                              Version     Available    Source
--------------------------------------------------------------------------------
AnyDesk          AnyDeskSoftwareGmbH.AnyDesk     ad 7.0.0    7.0.0        winget
```

Whoho this issue is back

Reply:
We're getting ready to push the first dev client with 1.2 in the version. we had some hiccups with sequencing. We needed to get an update to the 1.1 client based on a couple of telemetry issues with "import" not reporting things quite right, and the UI changes to the App Installer for Windows 10.

Reply:
I got the update long back but came to know after reading this message. 😂